src/opera/instance/node.py

```diff
@@ -55,7 +55,7 @@ def get_host(self):
         return self.template.get_host() or "localhost"
 
     def deploy(self):
-        print("  Deploying {}".format(self.tosca_id))
+        thread_utils.print_thread("  Deploying {}".format(self.tosca_id))
```

Comment:
I've noticed that in your implementation workers start with 0 (Worker_0, Worker_1). Is this ok or would it be more sensible if they begin with 1 like in @matejart's example in #51:

```console
[Worker-001] Deploying my-workstation_0
[Worker-001] Deploying hello-2_0
[Worker-002] Deploying hello-3_0
[Worker-001]   Executing create on hello-2_0
...
```

Reply:
This is a fair note, I was thinking about the same (also add padding number with leading 0 as Matej proposed). The problem is that my solution is simple and relies on thread naming that is provided by ThreadPoolExecutor, that automatically names threads with prefix adding number starting with 0. So in order to do that we would need to parse the name we get from ThreadPool, add 1, add padding and print. I was not sure this is worth it.

Reply:
While it would be prettier to have those number padded, I am happy enough with the thread names, so I wouldn't spend extra time on the formatting.